src/common/root.js

```diff
@@ -9,7 +9,21 @@ import store from './store';
 
 const history = syncHistoryWithStore(browserHistory, store);
 
+
 export default class Root extends Component {
+
+  componentWillMount() {
```

Comment:
It would be better for it to be `componentDidMount` to make sure it's in DOM already. `componentWillMount` is equivalent of doing it in constructor and (unnecessarily) called during server side rendering (https://reactjs.org/docs/react-component.html#componentwillmount).

Reply:
Sorry, I did know this but typo'ed it in. Problematically, using `componentDidMount` doesn't work because React doesn't like you mutating history at that point:

>Warning: [react-router] You cannot change \<Router history>; it will be ignored

Reply:
I stand corrected. Actually setting up a GitHub token and following the flow I can see that the behaviour is correct. Fixed as 5a1f736.